Source/WebCore/Modules/webcodecs/WebCodecsVideoDecoder.cpp

```diff
@@ -83,32 +83,41 @@ static bool isSupportedDecoderCodec(const String& codec, const Settings::Values&
 
 static bool isValidDecoderConfig(const WebCodecsVideoDecoderConfig& config)
```

Comment:
If we add comments, I prefer adding links to the spec(https://w3c.github.io/webcodecs/#valid-videodecoderconfig) than individual steps (spec can evolve, links will hopefully be more stable).